Homework 1/Lesson3.2/src/nekretnina/Kuca.java

```diff
@@ -0,0 +1,49 @@
+package nekretnina;
+
+import nekretnina.Nekretnina.Vlasnik;
+
+public class Kuca extends Nekretnina{
+
+	
+	public void Kuca() {
+		adress = "Milutina Milankovica 9dj";
+		zone = "zone 1";
+		kvadratura = 1234567890;
+		okucnica = 25;
+	}
+	
+	public void Kuca(String adress, String zone, int kvadratura, Vlasnik Vlasnik, double okucnica) {
+		this.adress = adress;
+		this.zone = zone;
+		this.kvadratura = kvadratura;
+		this.Vlasnik = Vlasnik;
+		this.okucnica = okucnica;
+		
+	}
+
+	
```

Comment:
koliko vidim nisi implementirala deo postavke gde se kaze da se cena racuna u zavisnosti od zone gde se kuca nalazi. to bi mogla ovako nekako da resis:
```
	public double izracunajCenu(double kvadratura, int zona, double povrsinaOkucnice) {
		double cena = 0.0;
		if (zona == 1) {
			cena = (kvadratura + povrsinaOkucnice) * 3000 * 0.15;
		} else if (zona == 2) {
			cena = (kvadratura + povrsinaOkucnice) * 2000 * 0.15;
		} else if (zona == 3) {
			cena = (kvadratura + povrsinaOkucnice) * 1000 * 0.15;
		} else {
			cena = (kvadratura + povrsinaOkucnice) * 500 * 0.15;
		}
		return cena;

	}
```
onda bi na primer mogla da dodas toString metodu za klasu Kuca:
```
	public String toString() {
		return "Adresa je: " + adresa + ", u zoni: " + zona + ", kvadrature: " + kvadratura + ", povrsina okucnice je: "
				+ povrsinaOkucnice + ", cena je: " + izracunajCenu(kvadratura, zona, povrsinaOkucnice) + ", "
				+ vlasnik.toString();

	}
```